signbank/dictionary/tests/test_adminviews.py

```diff
@@ -135,13 +135,18 @@ def test_get_ready_for_validation_csv(self):
         body = list(cvs_reader)
         self.assertEqual(len(body), 2)
 
+        video_tokens = GlossVideoToken.objects.filter(video=glossvid)
+        self.assertEqual(video_tokens.count(), 1)
+        video_token = video_tokens.get()
         headers = body[0]
         body = body[1]
```

Comment:
**suggestion**: use a different variable name for the complete list of rows from the csv, and the individual row being used for validation